### DISCUSSION TITLE
missing/ghots Icon

### DISCUSSION THREAD

**Discussion by lucasbcgeo** [AnubisNekhet/AnuPpuccin#222] [Snippets]
Looks like nothing is there, but take a look at the picture. What do i have to do appear again.[](url)
![fd](https://github.com/AnubisNekhet/AnuPpuccin/assets/143296622/8c2126e7-41cd-4a37-ac1e-eeaae53339a9)

**Comment by syrth-io** [comment]
hi! are you still having this problem (and if not, does it reappear after enabling the custom background snippet)? I have a couple hunches as to what might be causing it, but want to make sure it hasn't resolved already before I do any digging.

**Reply by lucasbcgeo** [reply]
Thnaks. Got the solution!
